README.md

```diff
@@ -56,83 +46,124 @@ Or manually checkout the submodule with `git submodule add git@github.com:Square
 
 ### Basic Initialization
 
+```swift
+let myValet = Valet.valet(with: Identifier(nonEmpty: "Druidia")!, accessibility: .whenUnlocked)
+```
+
 ```objc
-VALValet *myValet = [[VALValet alloc] initWithIdentifier:@"Druidia" accessibility:VALAccessibilityWhenUnlocked];
+VALValet *const myValet = [VALValet valetWithIdentifier:@"Druidia" accessibility:VALAccessibilityWhenUnlocked];
 ```
 
-To begin storing data securely using Valet, you need to create a VALValet instance with:
+To begin storing data securely using Valet, you need to create a Valet instance with:
 
-* An identifier – a string that is used to identify this Valet.
-* An accessibility value – an enum ([VALAccessibility](Valet/VALValet.h)) that defines when you will be able to store and retrieve data.
+* An identifier – a non-empty string that is used to identify this Valet. The Swift API uses an `Identifier` wrapper class to enforce the non-empty constraint.
+* An accessibility value – an enum ([Accessibility](Sources/Accessibility.swift#L25)) that defines when you will be able to persist and retrieve data.
```

Comment:
Thoughts on flipping the order between the text and the example? As it is, I see the identifier without knowing what it actually is, and then I need to scroll down to find out.

Reply:
@EricMuller22 thoughts on this? We originally put the example first to showcase how damned simple it was. But I agree we've traded some simplicity for better compile-time safety, which might make us want to swap here.